src/handlers/shared/start.ts

```diff
@@ -156,15 +156,17 @@ async function fetchUserIds(context: Context, username: string[]) {
 }
 
 async function handleTaskLimitChecks(username: string, context: Context, logger: Context["logger"], sender: string) {
-  const openedPullRequests = await getAvailableOpenedPullRequests(context, username);
+  const { approved, changes } = await getAvailableOpenedPullRequests(context, username);
   const assignedIssues = await getAssignedIssues(context, username);
   const { limit } = await getUserRoleAndTaskLimit(context, username);
 
+  const adjustedAssignedIssues = assignedIssues.length - approved.length + changes.length;
+
   // check for max and enforce max
-  if (Math.abs(assignedIssues.length - openedPullRequests.length) >= limit) {
+  if (Math.abs(adjustedAssignedIssues) >= limit) {
```

Comment:
if I have 1 issue and 1 PR with requested changes, it will count as 2 here but it should only count 1